README.md

```diff
@@ -135,21 +135,21 @@ for different rotation representations such as Quaternions, Euler angles or rota
 
 ## Structure and Modules of the CT
 
-The Control Toolbox consists of three main modules. The Core module (ct_core), the Optimal 
-Control (ct_optcon) module and the rigid body dynamics (ct_rbd) module. 
+The Control Toolbox consists of three main modules. The Core (`ct_core`) module, the Optimal 
+Control (`ct_optcon`) module and the rigid body dynamics (`ct_rbd`) module. 
```

Comment:
Thank you for fixing those typos!
If "Core" and "Optimal Control" are now capitalized, would it not make sense to do the same for "Rigid Body Dynamics" and Models?